caps/types.go

```diff
@@ -31,6 +31,9 @@ type Type struct {
 	// Name is a key that identifies the capability type. It must be unique
 	// within the whole OS. The name forms a part of the stable system API.
 	Name string
+	// RequiredAttrs contains names of attributes that are understood by
```

Comment:
In nitpick mode now (sorry for that!). Maybe instead of "that are understood by" "that are required by" ?